### PR TITLE
fix default values of helm chart enforcing pg ssl

### LINKED ISSUE #288: Helm chart working out of the box with postgres SSL (double check "prefer" setting)

### PR DESCRIPTION
**Pull request by vieiralucas**
Closes #288

### CODE REVIEW

File: chart/templates/api-deployment.yaml [hunk 1]
```diff
@@ -49,7 +49,7 @@ spec:
             - name: POSTGRES_DATABASE
               value: '{{ .Values.api.env.postgresDatabase }}'
             - name: POSTGRES_SSL_DISABLED
-              value: '{{ .Values.api.env.postgresSslDisabled | default "false" }}'
+              value: '{{ .Values.api.env.postgresSslDisabled }}'
```
Comment (lucasfcosta) [inline chart/templates/api-deployment.yaml:52]: makes sense that this was wrong given we were explicitly setting it to false

File: chart/templates/api-deployment.yaml [hunk 2]
```diff
@@ -94,9 +94,9 @@ spec:
             - name: FRONTEND_URL
               value: '{{ .Values.api.env.frontendUrl }}'
             - name: JUPYTER_HOST
-              value: 'briefer-jupyter-service'
+              value: '{{ .Values.api.env.jupyterHost | default "briefer-jupyter-service" }}'
             - name: JUPYTER_PORT
-              value: '{{ .Values.api.env.jupyterPort }}'
+              value: '{{ .Values.api.env.jupyterPort | default "8888" }}'
```
Comment (lucasfcosta) [inline chart/templates/api-deployment.yaml:99]: nice

File: chart/values.yaml [hunk 3]
```diff
@@ -97,18 +97,11 @@ api:
     frontendUrl: http://localhost:4000
     apiUrl: http://localhost:8080
     logLevel: info
-    jupyterRepository: docker.io
-    jupyterImage: briefer/enterprise/jupyter
-    jupyterTag: v566
-    jupyterPort: 8888
```
Comment (lucasfcosta) [inline chart/values.yaml:103]: We don't need this because the jupyter env is not created by the API on the OSS version, right? If so LGTM, just confirming

**Review by lucasfcosta — APPROVED** [review]

File: chart/templates/api-deployment.yaml (hunk 1 above)
Comment (vieiralucas) [reply chart/templates/api-deployment.yaml:52]: yeah, to use "prefer" it needs to be `null` neither `false` nor `true`

**Pull request merged**